Scripts/script-CommonServer.yml

```diff
@@ -136,10 +136,19 @@ script: |-
           mdResults = '### ' + name + '\n';
       }
       if (t && t.length) {
-          if (headers.length > 1) {
-              mdResults += headers.join('|') + '\n';
+          var newHeaders = [];
```

Comment:
I can't change header because it is used later as the keys of the original object. 
Anyway I changed it a bit so that it copies headers into newHeaders in any case.